### ISSUE TITLE
Upgrade to a 12-bit ADC

### ISSUE THREAD

**Issue by ezraboley** [badgerloop-software/podiv-altium#67]
**Is your feature request related to a problem? Please describe.**
Our resolution is preventing us from getting more accurate readings from sensors using our ADCs

**Describe the solution you'd like**
Replace the current 8-bit ADC with a 12-bit IC (ideally if there exists one with the same footprint)

**Describe alternatives you've considered**
A 10 bit ADC, or reworking the footprint on the next rev.

**Additional context**
Also need to do something about the wild Vref line

**Comment by rcastle1** [comment]
Agreed. I think the power distribution at the board level should be reference VBus with LDOs just before all of the sensitive analog circuitry. You can get precision reference voltage ICs as well. Additionally, utilize hardware low pass filters
